src/Network/Gitit/Layout.hs

```diff
@@ -132,23 +135,24 @@ linkForTab tabli base' page rev ViewTab =
                       else s
   in if isDiscussPage page
         then tabli DiscussTab << anchor !
-              [href $ base' ++ urlForPage (origPage page)] << "page"
+              [href $ base' ++ urlForPage (origPage page), accesskey 'c'] << "page"
         else tabli ViewTab << anchor !
               [href $ base' ++ urlForPage page ++
                       case rev of
                            Just r  -> "?revision=" ++ r
-                           Nothing -> ""] << "view"
+                           Nothing -> "", accesskey 'c'] << "view"
```

Comment:
In the current implementation (as of commit 94cb5a49b) when looking at a revision of a page, the <kbd>c</kbd> accesskey is useless. There is no way to go from a revision to the current/latest version of the page, which MediaWiki supports. For example, on https://en.wikipedia.org/w/index.php?title=Human&oldid=1128628188 pressing the accesskey combination with <kbd>c</kbd> navigates to https://en.wikipedia.org/wiki/Human

After using this for a bit, I think it would be better to mimic MediaWiki and move `accesskey="c"` to a link, which would always go to the current version of the page, i.e. something along these lines:

```diff
diff --git a/data/templates/content.st b/data/templates/content.st
index 0d1229d..9c39d64 100644
--- a/data/templates/content.st
+++ b/data/templates/content.st
@@ -2,7 +2,7 @@
   $if(revision)$
     <h2 class="revision">Revision $revision$ (click the page title to view the current version)</h2>
   $endif$
-  <h1 class="pageTitle"><a href="$base$$pageUrl$">$pagetitle$</a></h1>
+  <h1 class="pageTitle"><a href="$base$$pageUrl$" accesskey="c">$pagetitle$</a></h1>
   <div class="subpages"><a href="../">&lt; parent directory</a>
     <a href="$base$$pageUrl$/">&gt; this directory</a>
   </div>
```

Reply:
This, however, would not work for switching from discussion pages (e.g. `@Sandbox`) to content pages. :slightly_frowning_face: